push_service/app.py

```diff
@@ -323,22 +323,28 @@ def post(self, target_id):  # pylint:disable=no-self-use
         target_id = decode_base64_id(target_id)
         return publish(data, target_id)
 
+# Check for path prefix
+if os.environ.get('PATH_PREFIX'):
+    PATH_PREFIX = "/" + os.environ.get('PATH_PREFIX')
```

Comment:
This would be more failsafe if this would strip leading and tailing slashes, i.e.,

```
os.environ.get("PATH_PREFIX").strip("/")
```